### PR TITLE
CAN-664

### PR DESCRIPTION
**Pull request by vikasranatalentelgia**
Add Support: Implement screen design for Add Support

### CODE REVIEW

File: src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx [hunk 1]
```diff
@@ -17,12 +17,52 @@ const ManageSupportUI = () => {
           </div>
         }
       >
+        <span className={styles.warning}>
+          <strong> Warning! </strong>`Agreement` is a parent camp to the list of
```
Comment (akuks) [inline src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx:36]: It should be from the common message file

File: src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx [hunk 2]
```diff
@@ -17,12 +17,52 @@ const ManageSupportUI = () => {
...
+            </div>
+            <Link href="">
+              <a>2 . Home Page Design Talentelgia</a>
+            </Link>
+          </Tag>
+        </Col>
+        <div className={styles.hrtag}></div>
         <div className={styles.notes}>
           {" "}
           Note : To change support order of camp, drag & drop the camp box on
```
Comment (akuks) [inline src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx:64]: Same as before

**Review by akuks — CHANGES_REQUESTED** [review]

File: src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx (hunk 1 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx:36]: It is a static design implementation for now. will be dynamic once API will be implemented.

File: src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx (hunk 2 above)
Comment (vikasranatalentelgia) [reply src/components/ComponentPages/ManageSupport/ManageSupportUI/index.tsx:64]: It is a static design implementation for now. will be dynamic once API will be implemented.

**Review by akuks — APPROVED** [review]

**Pull request merged**